atlasdb-cassandra/src/main/java/com/palantir/atlasdb/keyvalue/cassandra/CassandraClientPool.java

```diff
@@ -132,7 +132,16 @@ public String toString() {
         }
     }
 
+    @VisibleForTesting
+    static CassandraClientPool createForTesting(CassandraKeyValueServiceConfig config) {
```

Comment:
Also I'm wondering if we can make this more descriptive of what it actually does, because when I see createForTesting I have no concept of the difference between this and regular.